internals/secrethub/audit_repo_test.go

```diff
@@ -102,9 +113,14 @@ func TestAuditRepoCommand_run(t *testing.T) {
 						},
 					}, nil
 				},
+				format:  formatTable,
 				perPage: 20,
+				terminalWidth: func(int) (int, error) {
+					return 83, nil
+				},
 			},
 			err: testError,
+			out: "",
```

Comment:
The table header isn't printed anymore in case there is an error before printing the first entry or there are no entries.